masschallenge/impact-api
# [AC-7217] expose version number to all apps

### Changes introduced in: [AC-7217](https://masschallenge.atlassian.net/browse/AC-7217):
- creates a release number as an environment variable

### How to test
on this branch `AC-7217`
- delete the docker image for `impact-api`
- run `docker rmi impact-api_web`
- run `make run-detached-server`
- run `docker exec -it impact-api_web_1 /bin/bash` to access the impact-api container
- inside the container's terminal run `printenv | grep ACCELERATE_VERSION`
- this should print the latest release tag
